neuromation/api/jobs.py

```diff
@@ -623,3 +635,17 @@ def inner() -> None:
 
     loop = asyncio.get_event_loop()
     await loop.run_in_executor(None, inner)
+
+
+class _AwaitDisconnection:
+    __slots__ = ("_proc",)
+
+    def __init__(self, proc: Process) -> None:
+        self._proc = proc
+
+    async def _wait(self) -> None:
+        # the tranpoline is needed to shift the return type from int to None
+        self._proc.wait()
```

Comment:
```suggestion
        await self._proc.wait()
```
because warning here:
>/home/shagren/PycharmProjects/platform-client-python/neuromation/api/jobs.py:648: RuntimeWarning: coroutine 'Process.wait' was never awaited
  self._proc.wait()
RuntimeWarning: Enable tracemalloc to get the object allocation traceback

Reply:
Thanks!